content/en/docs/tasks/traffic-management/egress/wildcard-egress-hosts/index.md

```diff
@@ -664,21 +618,20 @@ The SNI proxy will forward the traffic to port `443`.
     {{< text bash >}}
     $ kubectl delete serviceentry sni-proxy
     $ kubectl delete destinationrule disable-mtls-for-sni-proxy
-    $ kubectl delete -f ./istio-egressgateway-with-sni-proxy.yaml
+    $ kubectl delete IstioOperator istio-egressgateway-with-sni-proxy -n istio-system
```

Comment:
This will delete the whole control plane, I would guess. Should be cleaned differently.